src/test/aztec/base/BridgeTestBase.sol

```diff
@@ -23,9 +23,9 @@ abstract contract BridgeTestBase is Test {
     error INVALID_ASSET_ADDRESS();
     error INVALID_LINKED_TOKEN_ADDRESS();
     error INVALID_LINKED_BRIDGE_ADDRESS();
-    error INVALID_BRIDGE_ID();
+    error INVALID_BRIDGE_CALL_DATA();
     error INVALID_BRIDGE_ADDRESS();
-    error BRIDGE_ID_IS_INCONSISTENT();
```

Comment:
Not sure you want to change these to something else than the deployed version. If getting the `INVALID_BRIDGE_ID()` error, it won't be able to match with an error from here, so you might have issues debugging it.

Reply:
Ok, I will leave it there for now and add there a TODO.

Reply:
Addressed in a5ac4a5e5c924e726777869d08320eed13b5c9fa